scripts/attach_ebs_cache.sh

```diff
@@ -11,8 +11,15 @@ INSTANCE_ID=$(curl http://169.254.169.254/latest/meta-data/instance-id)
 # TODO also mount various other aspects of docker image metadata
 
 # Check for existing mount, assume we can continue if existing
```

Comment:
```suggestion
# Check for existing mount, stop early if it exists
```

Reply:
hmm this isn't true though. We don't stop earthly

Reply:
if this is true, the spot is still running, we run earthly bootstrap still but it quickly reports typically earthly is still running